src/webapp/forms.py

```diff
@@ -301,3 +302,145 @@ def get_yaml(self) -> str:
+        ('Earth Sciences', 'Earth Sciences'),
+        ('Ecological and Environmental Sciences', 'Ecological and Environmental Sciences'),
+        ('Economics', 'Economics'),
+        ('Education', 'Education'),
+        ('Educational Psychology', 'Educational Psychology'),
+        ('Elementary Particles', 'Elementary Particles'),
+        ('Engineering', 'Engineering'),
+        ('Evolutionary Biology', 'Evolutionary Biology'),
+        ('Evolutionary Sciences', 'Evolutionary Sciences'),
+        ('Finance', 'Finance'),
+        ('Fluid Dynamics', 'Fluid Dynamics'),
+        ('Genetics and Nucleic Acids', 'Genetics and Nucleic Acids'),
+        ('Genomics', 'Genomics'),
+        ('Geographic Information Science', 'Geographic Information Science'),
+        ('Geography', 'Geography'),
+        ('Geological and Earth Sciences', 'Geological and Earth Sciences'),
+        ('Gravitational Physics', 'Gravitational Physics'),
+        ('High Energy Physics', 'High Energy Physics'),
+        ('Information Theory', 'Information Theory'),
+        ('Information, Robotics, and Intelligent Systems', 'Information, Robotics, and Intelligent Systems'),
+        ('Infrastructure Development', 'Infrastructure Development'),
+        ('Logic', 'Logic'),
+        ('Materials Research', 'Materials Research'),
+        ('Materials Science', 'Materials Science'),
+        ('Mathematical Sciences', 'Mathematical Sciences'),
+        ('Mathematics', 'Mathematics'),
+        ('Medical Imaging', 'Medical Imaging'),
+        ('Medical Sciences', 'Medical Sciences'),
+        ('Microbiology', 'Microbiology'),
+        ('Molecular and Structural Biosciences', 'Molecular and Structural Biosciences'),
+        ('Multi-Science Community', 'Multi-Science Community'),
+        ('Multidisciplinary', 'Multidisciplinary'),
+        ('Nanoelectronics', 'Nanoelectronics'),
+        ('National Laboratory', 'National Laboratory'),
+        ('Network Science', 'Network Science'),
+        ('Neuroscience', 'Neuroscience'),
+        ('Nuclear Physics', 'Nuclear Physics'),
+        ('Nutritional Science', 'Nutritional Science'),
+        ('Ocean Sciences', 'Ocean Sciences'),
+        ('Other', 'Other'),
+        ('Particle Physics', 'Particle Physics'),
+        ('Physical Chemistry', 'Physical Chemistry'),
+        ('Physical Therapy', 'Physical Therapy'),
+        ('Physics', 'Physics'),
+        ('Physics and astronomy', 'Physics and astronomy'),
+        ('Physiology', 'Physiology'),
+        ('Planetary Astronomy', 'Planetary Astronomy'),
+        ('Plant Biology', 'Plant Biology'),
+        ('Research Computing', 'Research Computing'),
+        ('Statistics', 'Statistics'),
+        ('Technology', 'Technology'),
+        ('Training', 'Training'),
+        ('Zoology', 'Zoology')
+    ])
+
+    description = TextAreaField(None, render_kw={
+        "style": "font-family:monospace; font-size:small;",
+        "rows": "5"
+    })
```

Comment:
Oof, can you generate this from the file on disk instead?

Reply:
Good call